### DISCUSSION TITLE
How should set validation_batch_size??

### DISCUSSION THREAD

**Discussion by jungsdao** [ACEsuit/mace#163] [Q&A]
Hello, 
I'd like to ask if there are some heuristics to set `validation_batch_size` ?
Normally validation set size is way smaller than training set. So having the same value for `batch_size` might be sensible. 
Could you give me some advice on this?
Thank you in advance!

**Comment by ilyes319** [comment]
The largest number that fits your GPU is the best. Something like 16 or 32 is a good start.
